src/services/measure/measure.js

```diff
@@ -0,0 +1,278 @@
+  this.sketchFeature = evt.feature;
+  this.overlay.getFeatures().clear();
+  this.createMeasureTooltip_();
+};
+
+
+/**
+ * Handle draw interaction `drawend` event.
+ * @param {ol.DrawEvent} evt
+ * @private
+ */
+ngeo.interaction.Measure.prototype.onDrawEnd_ = function(evt) {
+  goog.dom.classlist.add(this.measureTooltipElement_, 'tooltip-static');
+  this.measureTooltip_.setOffset([0, -7]);
+  this.sketchFeature = null;
+};
+
+
+/**
+ * Creates a new help tooltip
+ * @private
+ */
+ngeo.interaction.Measure.prototype.createHelpTooltip_ = function() {
+  this.removeHelpTooltip_();
+  this.helpTooltipElement_ = goog.dom.createDom(goog.dom.TagName.DIV);
+  goog.dom.classlist.add(this.helpTooltipElement_, 'tooltip');
+  this.helpTooltip_ = new ol.Overlay({
+    element: this.helpTooltipElement_,
+    offset: [15, 0],
+    positioning: 'center-left'
+  });
+  this.getMap().addOverlay(this.helpTooltip_);
+};
+
+
+/**
+ * Destroy the help tooltip
+ * @private
+ */
+ngeo.interaction.Measure.prototype.removeHelpTooltip_ = function() {
+  this.getMap().removeOverlay(this.helpTooltip_);
+  if (!goog.isNull(this.helpTooltipElement_)) {
+    this.helpTooltipElement_.parentNode.removeChild(this.helpTooltipElement_);
+  }
+  this.helpTooltipElement_ = null;
+  this.helpTooltip_ = null;
+};
+
+
+/**
+ * Creates a new measure tooltip
+ * @private
+ */
+ngeo.interaction.Measure.prototype.createMeasureTooltip_ = function() {
+  this.removeMeasureTooltip_();
+  this.measureTooltipElement_ = goog.dom.createDom(goog.dom.TagName.DIV);
+  goog.dom.classlist.addAll(this.measureTooltipElement_,
+      ['tooltip', 'tooltip-measure']);
+  this.measureTooltip_ = new ol.Overlay({
+    element: this.measureTooltipElement_,
+    offset: [0, -15],
+    positioning: 'bottom-center'
+  });
+  this.getMap().addOverlay(this.measureTooltip_);
+};
+
+
+/**
+ * Destroy the help tooltip
+ * @private
+ */
+ngeo.interaction.Measure.prototype.removeMeasureTooltip_ = function() {
+  if (!goog.isNull(this.measureTooltipElement_)) {
+    this.measureTooltipElement_.parentNode.removeChild(
+        this.measureTooltipElement_);
+    this.measureTooltipElement_ = null;
+    this.measureTooltip_ = null;
+  }
+};
+
+
+/**
+ * @inheritDoc
+ */
+ngeo.interaction.Measure.prototype.setActive = function(active) {
+  goog.base(this, 'setActive', active);
+  if (!goog.isNull(this.drawInteraction)) {
+    this.drawInteraction.setActive(active);
+    if (!this.getMap()) {
+      return;
+    }
+    if (active) {
+      this.createMeasureTooltip_();
+      this.createHelpTooltip_();
+    } else {
+      this.overlay.getFeatures().clear();
+      this.getMap().removeOverlay(this.measureTooltip_);
+      this.removeMeasureTooltip_();
+      this.removeHelpTooltip_();
+    }
+  }
+};
+
+
+/**
+ * Handle pointer move.
+ * @param {ol.MapBrowserEvent} evt
+ * @private
+ */
+ngeo.interaction.Measure.prototype.handlePointerMoveEvent_ = function(evt) {
+  if (!this.getActive()) {
```

Comment:
I guess this case cannot occur. If the interaction is not active then handlers are not called.